test/kv/client.go

```diff
@@ -0,0 +1,64 @@
+package kv
+
+import (
+	"fmt"
+	"net/rpc"
+	"os"
+)
+
+type StoreClient struct {
+	client *rpc.Client
+}
+
+func NewStoreClient(network string, address string) (*StoreClient, error) {
+	client, err := rpc.DialHTTP(network, address)
+	if err != nil {
+		return nil, err
+	}
+
+	return &StoreClient{
+		client: client,
+	}, nil
+}
+
+func (s *StoreClient) Set(key, val string) error {
```

Comment:
Yet another crazy idea... maybe crazier than the others. Would something like badgerDB (https://godoc.org/github.com/alash3al/redix/kvstore/badgerdb, https://dbdb.io/db/badgerdb) be useful here as a K/V store? It's intended more for embedded K/V store... I don't think it comes with a REST API, but I don't think we absolutely need that here? It's implemented in Go, has basic Get/Set, and iterators, which might come in handy someday.

Reply:
I saw this before and thought that it's a bit overkill. For now something simple should suffice. It is worthwhile though to use interfaces so that if we decide to minimise any hurt from switching out the backing store.

Reply:
Yeah, badgerdb would be a bit heavy-handed for what we need.